.github/workflows/docker-image-arm.yml

```diff
@@ -13,19 +13,19 @@ concurrency:
   group: ${{ github.ref }}-arm
   cancel-in-progress: true
 
+permissions:
```

Comment:
This allows the OIDC token exchange with Depot for a single-use token so that you don't need static access tokens in your secrets.